hud-data/weirdcorehud.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "WeirdcoreHUD",
+  "author": "Tomma",
+  "contributors": [
+    "Yara"
+  ],
+  "releaseDate": "2024 04 06",
+  "resources": [
+    "https://youtu.be/hlAn1QJPX7o"
```

Comment:
To your resources, you'd also want to add the image you included (`sfondo-hud`). You'd probably want to list it before the youtube link as well

Reply:
fixed, thx  for warning me